run-bas.sh

```diff
@@ -1,12 +1,15 @@
 #!/usr/bin/env bash
 
+set -euo pipefail
+
 # go to directory where this file is located
 cd "$(dirname "$0")"
 
-: "${VX_CONFIG_ROOT:="./config"}"
+: "${VX_CONFIG_ROOT:="/vx-config"}"
```

Comment:
might this line no longer be needed?

Reply:
I could remove this since it's set in production by the `.service` files. It's probably worth putting in an explicit check for it, though, for testing. What do you think?

Reply:
I'm not seeing used at all in this file, though. Are you setting it for the other scripts we're sourcing? Wouldn't those other scripts be better positioned to set a default?

Reply:
You're right. Will remove.